frontend/src/v4/routes/viewerGui/components/previewDetails/previewDetails.styles.ts

```diff
@@ -108,6 +109,8 @@ export const Header = styled(AccordionSummary).attrs({
 	}
 ` as any;
 
+export const Grid = styled(GridBase)``;
```

Comment:
Any reason why you removed the import of MUI Grid from the components file andput an import for GridBase here instead?